src/views/RecordingPreviewer/RecordingPreviewer.tsx

```diff
@@ -118,6 +150,17 @@ export function RecordingPreviewer() {
               </IconButton>
             </DropdownMenu.Trigger>
             <DropdownMenu.Content>
+              {settings?.recorder.enableBrowserRecorder && (
```

Comment:
Having it in the dropdown is not the prettiest, but it doesn't really matter right now. I will be collecting feedback on the UI once we can record something of value.